docs/server/cache.rst

```diff
@@ -47,81 +41,11 @@ Cache Backends
 --------------
 
 Django supports |multiple cache backends|_ (methods of storing cache data).
-You can specify which backend to use by overriding the value of
-:setting:`CACHES` in your configuration file.
-
-
-.. _cache#memcached:
-
-Memcached
-^^^^^^^^^
-
-::
-
-    CACHES = {
-        'default': {
-            'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
-            'LOCATION': '127.0.0.1:11211',
-        }
-    }
-
-:ref:`Memcached <django:memcached>` is the **recommended cache backend**, it
-provides the best performance.  And works fine with multiprocessing servers
-like Apache. It requires the `python-memcached` package and a running
-memcached server. Due to extra dependencies it is not enabled by default.
-
-
-.. _cache#memcached_on_unix_sockets:
-
-Memcached on Unix sockets
-"""""""""""""""""""""""""
-
-::
-
-    CACHES = {
-        'default': {
-            'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
-            'LOCATION': 'unix:/path/to/memcached.sock',
-        }
-    }
-
-If you don't want Pootle using TCP/IP to access memcached then you can use Unix
-sockets.  This is often a situation in hardened installations using SELinux.
+However, Redis is the only cache backend supported by Pootle.  We use some
+custom features of Redis so cannot support other backends. You can customise
+the Redis cache settings by overriding the value of :setting:`CACHES` in your
+configuration file.
```

Comment:
Might be worth mentioning there's an example configuration in *90-local.conf.sample*.